lib/overhang.js

```diff
@@ -44,6 +44,9 @@ $.fn.overhang = function (arguments) {
     easing       : "easeOutBounce",
     html         : false,
     overlay      : false,
+    multiline    : false,
+    textWidth    : "250px",
+    textHeight   : "30px",
```

Comment:
A default height a little higher would make sense for a textarea (so it is more easily distinguished from an input field.) Something around 100px would be better.